app/_data/docs_nav_kuma_dev.yml

```diff
@@ -565,88 +552,21 @@ items:
-            url: "/reference/http-api/#dataplanes"
-          - text: Dataplane Overviews
-            url: "/reference/http-api/#dataplane-overviews"
-          - text: Health Check
-            url: "/reference/http-api/#health-check"
-          - text: Proxy Template
-            url: "/reference/http-api/#proxy-template"
-          - text: Traffic Permission
-            url: "/reference/http-api/#traffic-permission"
-          - text: Traffic Log
-            url: "/reference/http-api/#traffic-log"
-          - text: Traffic Route
-            url: "/reference/http-api/#traffic-route"
-          - text: Traffic Trace
-            url: "/reference/http-api/#traffic-trace"
-          - text: Fault Injection
-            url: "/reference/http-api/#fault-injection"
-          - text: Retry
-            url: "/reference/http-api/#retry"
-          - text: Timeout
-            url: "/reference/http-api/#timeout"
-          - text: Zones
-            url: "/reference/http-api/#zones"
-          - text: Zone Overview
-            url: "/reference/http-api/#zone-overview"
-          - text: Zone Ingresses
-            url: "/reference/http-api/#zone-ingresses"
-          - text: Zone Ingress Overviews
-            url: "/reference/http-api/#zone-ingress-overviews"
-          - text: Zone Egresses
-            url: "/reference/http-api/#zone-egresses"
-          - text: Zone Egress Overviews
-            url: "/reference/http-api/#zone-egress-overviews"
-          - text: External Services
-            url: "/reference/http-api/#external-services"
-          - text: Service Insights
-            url: "/reference/http-api/#service-insights"
-          - text: Secrets
-            url: "/reference/http-api/#secrets"
-          - text: Global Secrets
-            url: "/reference/http-api/#global-secrets"
-          - text: Multi-zone
-            url: "/reference/http-api/#multi-zone"
-          - text: Dataplane Proxy Tokens
-            url: "/reference/http-api/#dataplane-proxy-tokens"
-          - text: Zone Ingress Tokens
-            url: "/reference/http-api/#zone-ingress-tokens"
-          - text: Global Insights
-            url: "/reference/http-api/#global-insights"
-          - text: Inspect API
-            url: "/reference/http-api/#inspect-api"
-      - text: Annotations and labels in Kubernetes mode
+      - text: Kubernetes annotations and labels
         url: /reference/kubernetes-annotations/
-        items:
-          - text: Labels
-            url: "/reference/kubernetes-annotations/#labels"
-          - text: Annotations
-            url: "/reference/kubernetes-annotations/#annotations"
       - text: Kuma data collection
         url: /reference/data-collection/
-        items:
-          - text: Enabling data collection
-            url: "/reference/data-collection/#enabling-data-collection"
-          - text: What data is collected
-            url: "/reference/data-collection/#what-data-is-collected"
+      - # Auto-generated
```

Comment:
Is this entry auto-generated or the underlying page?

Reply:
@slonka I believe that just indicates that the pages below are autogenerated, the entry itself was manually entered.